mlflow/genai/optimize/base.py

```diff
@@ -55,13 +55,13 @@ def optimize_prompt(
 
             - inputs (required): A column containing single inputs in dict format.
               Each input should contain keys matching the variables in the prompt template.
-            - expectations (optional): A column containing a dictionary
+            - expectations (required): A column containing a dictionary
```

Comment:
```suggestion
            - expectations: A column containing a dictionary
```

do we actually need this?

Reply:
Actually, Let's remove them.